low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -0,0 +1,337 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+#################################################################
+# This script performs following operations
+# - Check if product is one of the enabled products
+# - Configures role in sspl.conf if supplied
+# - Executes sspl_reinit script
+# - Updates cluster nodes passed in cli to consul
+#################################################################
+
+import os
+import sys
+import subprocess
+import shutil
+import psutil
+import json
+import re
+
+# sys.path.insert(0, '/opt/sumedh/cortx-sspl/low-level/')
+
+import dbus
+
+from cortx.sspl.lowlevel.framework.base import sspl_constants as consts
+from cortx.sspl.lowlevel.framework.utils.salt_util import SaltInterface
+from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_setup import Cmd as SSPLSetup
```

Comment:
sspl_setup is the base script to call this sspl_reset script.
In this case, when we import this sspl_reset in sspl_setup, this will create circular dependency. So find a way to avoid this.